interactive_engine/executor/ir/core/src/plan/logical.rs

```diff
@@ -1502,7 +1504,7 @@ fn is_params_all_labels(params: &pb::QueryParams) -> bool {
             .as_ref()
             .and_then(|store_meta| store_meta.schema.as_ref())
         {
-            let params_label_ids: BTreeSet<LabelId> = params
+            let params_label_ids: Vec<LabelId> = params
```

Comment:
Seems params will carry duplicated tables in Gremlin for now. e.g., `g.V().hasLabel("PERSON","PERSON","SOFTWARE")`.

Reply:
Fixed.